languages/ruby/bitwarden_sdk/lib/extended_schemas/schemas.rb

```diff
@@ -0,0 +1,64 @@
+
+module BitwardenSDK
+    class SelectiveCommand < Command
+      attribute :password_login,     PasswordLoginRequest.optional.default(nil)
+      attribute :api_key_login,      APIKeyLoginRequest.optional.default(nil)
+      attribute :access_token_login, AccessTokenLoginRequest.optional.default(nil)
+      attribute :get_user_api_key,   SecretVerificationRequest.optional.default(nil)
+      attribute :fingerprint,        FingerprintRequest.optional.default(nil)
+      attribute :sync,               SyncRequest.optional.default(nil)
```

Comment:
Is it possible to remove the unused commands? Or do they need to exist.

```suggestion
      attribute :access_token_login, AccessTokenLoginRequest.optional.default(nil)
```

Reply:
there is a problem when inheriting from Command class, which is a dry struct in Ruby, enforcing the structure. Wee would need to recreate own version of Command in order to bypass this.